////////////////////////////////////////////////////////////////////////////////////////////////////
///
/// @file      ConfigurationStatus.proto
///
/// @brief     This Protocol Buffer contains configuration status information to persist in non-
///            volatile memory.
///
/// @attention Copyright (C) 2017 Bose Corporation All Rights Reserved
///
///            Bose Corporation
///            The Mountain Road,
///            Framingham, MA 01701-9168
///            U.S.A.
///
///            This program may not be reproduced, in whole or in part, in any form by any means
///            whatsoever without the written permission of Bose Corporation.
///
////////////////////////////////////////////////////////////////////////////////////////////////////

package ProductPb;

message ConfigurationStatus
{
    message Status
    {
        optional bool network  = 1 [default = false];
        optional bool language = 2 [default = false];
        optional bool account  = 3 [default = false];
    }

    optional Status status = 1;
}

////////////////////////////////////////////////////////////////////////////////////////////////////
///                                        End of File                                           ///
////////////////////////////////////////////////////////////////////////////////////////////////////
